src/Sylius/Bundle/CoreBundle/Resources/config/test_services.xml

```diff
@@ -23,6 +23,7 @@
         <parameter key="sylius.random_invoice_number_generator.class">Sylius\Component\Core\Test\Services\RandomInvoiceNumberGenerator</parameter>
         <parameter key="sylius.behat.factory.default_france_channel.class">Sylius\Component\Core\Test\Services\DefaultFranceChannelFactory</parameter>
         <parameter key="sylius.behat.factory.default_channel.class">Sylius\Component\Core\Test\Services\DefaultChannelFactory</parameter>
+        <parameter key="sylius.behat.email_checker.class">Sylius\Component\Core\Test\Services\EmailChecker</parameter>
```

Comment:
I'm not 100% sure about this change. It could stay in a previous file until we would have a use case in another place, or at least we should not add a behat key word. Still, I would vote for leaving it in a previous place.